src/Storages/ExternalStream/ExternalStreamCounter.h

```diff
@@ -31,6 +35,8 @@ class ExternalStreamCounter
             {"WriteBytes", write_bytes.load()},
             {"WriteCounts", write_counts.load()},
             {"WriteFailed", write_failed.load()},
+            {"MessagesBySize", messages_by_size.load()},
```

Comment:
we probably don't need this in counters since the counters is basically for metering

Reply:
>  they are dupliated to WriteCounts / WriteBytes

`MessagesBySize` indicates that the number of messages generated because of it reached the max message size limit, and `MessagesByRow`indicates that the number of messages generated because of it reached the max messager row limit. So, in a normal case that when the whole Chunk can fit into a message without hitting those limits, then these two counters won't increase.

So unless you have other concerns, I will keep these. Since we don't print logs for generating messages, these will be useful for monitoring how the sinks work.